README.md

```diff
@@ -1,3 +1,125 @@
 # hyp3
 ![Static code analysis](https://github.com/ASFHyP3/hyp3/workflows/Static%20code%20analysis/badge.svg)
 ![Deploy to AWS](https://github.com/ASFHyP3/hyp3/workflows/Deploy%20to%20AWS/badge.svg)
+![Run tests](https://github.com/ASFHyP3/hyp3/workflows/Run%20tests/badge.svg)
+
+A processing environment for running RTC Gamma container jobs in Amazon Web Services.
+
+## Table of contents
+- [Deployment](#deployment)
+  - [Prerequisites](#prerequisites)
+  - [Stack Parameters](#stack-parameters)
+  - [Deploy with Cloudformation](#deploy-with-cloudformation)
+- [Testing](#testing)
+- [Running the API Locally](#running-the-api-locally)
+## Deployment
+
+
+### Prerequisites
+These resources are required for a successful deployment, but managed separately:
+
+- IAM role configured for api gateway access logging
+- IAM user and roles for automated cloudformation deployments
+- RTC Gamma container image repository and tag
+- S3 bucket for cloudformation deployment artifacts
+- DNS record for custom api domain name
+- SSL certificate in AWS Certificate Manager for custom api domain name
+- Earthdata Login account authorized to download data from ASF
+- default VPC
+
+### Stack Parameters
```

Comment:
should consider moving these descriptions into the cloudformation.yaml itself, then have the readme just say "review the stack parameters" and link to the template